lib/pipeline/addon/services/pipeline-github.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
```

Comment:
The github hooks for watching code and such are moving soon so coordinate with @westlywright.

Reply:
Have no idea what to do for now.

Reply:
@biblesyme I talked to Vince and understand the issue now. We've changed the way github authorization works so this will need to be updated to use the new logic for auth. I'm working on the auth changes as we speak but prob have a few more hours before the pr is ready. Let me get that in and then we can sync up to talk about the changes

Reply:
@westlywright ok, no hurry. just let me know when you are ready.